### DISCUSSION TITLE
Github thinks I have an unchecked notification even though I don't

### DISCUSSION THREAD

**Discussion by Maximkaaa** [community/community#124214] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello dear Github team.

For the last year Github says that I have an unread notification even though I don't have any. There is a blue dot next to the notifications icon. Notifications page says I have 1 unread notification, 1 "Participating" and 1 "Mentioned". But with and without any filters there are no notifications in the list. Nothing I have trying neither shows nor removes this hidden notification.

I've been waiting for a year for this issue to go away by itself, but since it didn't I hope for your help.

**Comment by ebndev** [comment]
Hi @Maximkaaa, thanks for being a part of the GitHub Community!

Please see [this previous discussion](https://github.com/orgs/community/discussions/123129#discussioncomment-9451090) for possible solutions.

Feel free to ask questions on the original threads if you run into an issue.
